Update masked entry styling and map breaking changes. (#970)
telerik/maui-docs: controls/map/selection.md

# .NET MAUI Map Selection

The Map allows users to select one or many shapes out of the source applied by each `MapShapefileLayer`. This feature provides both visual and programmatic feedback for the actions of the user. 

The following members of the [ShapefileLayer]({% slug map-layers-shapefilelayer%}) class are related to the selection: 

> * `SelectionMode` (`enum` of type `MapSelectionMode`)&mdash;Indicates what will be the selection. The options are:

Reviewer: ```suggestion
* `SelectionMode` (`enum` of type `MapSelectionMode`)&mdash;Indicates the selection mode. The options are:
```

	* `None`
	* `Single`
	* `Multiple`

* `SelectedShape` (of type `IShape`)&mdash;Defines the currently selected shape instance. When multiple selection is enabled, this value is set to the first selected shape. 

> * `SelectedShapes` (read-only collection of type `ObservableCollection&lt;IShape&gt;`)&mdash;Reads the collection with the currently selected shapes. When the selection is single only one shape can be selected – thus the collection will have count = 1. 

Reviewer: ```suggestion
* `SelectedShapes` (read-only collection of type `ObservableCollection&lt;IShape&gt;`)&mdash;Reads the collection with the currently selected shapes. When the selection is `Single`, only one shape can be selected–thus the collection will have count = 1. 
```
